assets/src/edit-story/elements/index.js

```diff
@@ -43,12 +44,33 @@ export const createNewElement = (type, attributes = {}) => {
   };
 };
 
-export const createPage = (attributes) => createNewElement('page', attributes);
+export const createPage = (attributes = {}) => {
+  const { elements, backgroundElementId } = attributes;
+  // Enforce having background element for each Page.
+  if (!backgroundElementId) {
+    const props = {
+      x: PAGE_WIDTH / 4,
+      y: PAGE_HEIGHT / 4,
+      width: PAGE_WIDTH / 3,
+      height: PAGE_HEIGHT / 3,
```

Comment:
What's with the magic numbers here? Why even give it a position and size, if it can only be the background anyway?

Reply:
Well, it has to have some kind of position and width-height!

I suppose any number is good, could change it to `1`. 🤷

Reply:
(Took these numbers from what's used for migration -- but it's true that it doesn't really matter.)

Reply:
Then set them to full page size (`x=y=0`, `width=PAGE_WIDTH`, `height=PAGE_HEIGHT`), that makes the most sense,  I guess? Even though they're of course completely ignored.

Even better would be [conditional prop types (with a custom validator)](https://reactjs.org/docs/typechecking-with-proptypes.html#proptypes), that doesn't require `x`, `y`, `width`, and `height` for elements with `isBackground=true`. It's however probably too much for this ticket.

Reply:
Well, for the sake of not having to import the page constants, changed all of these to `1` and added a comment mentioning that the value is irrelevant.

IDK if there's any point changing the values to what makes the most sense if they're ignored anyway :)

Reply:
```js
  x: "Zoidberg"
```

![zoidberg](https://user-images.githubusercontent.com/637548/78934083-53464600-7a78-11ea-8d0c-14cddd0c1c40.jpg)